modules/file-integrity-node-status-success.adoc

```diff
@@ -3,7 +3,7 @@
 // * security/file_integrity_operator/file-integrity-operator-understanding.adoc
 
 [id="file-integrity-node-status-success_{context}"]
-= `FileIntegrityNodeStatus` success status example
+= File Integrity node status success example
```

Comment:
Why update line 6? `FileIntegrityNodeStatus` is more reasonable. It is a CR name.